### ISSUE TITLE
Lambda deployment with Terraform fails

### ISSUE THREAD

**Issue by ciaranevans** [localstack/localstack#1818]
Creating just as a link, as @whummer has already got eyes on this from Slack.

As of version`2.40` of the Terraform `aws` provider, Lambda deployments fail.

See: https://github.com/terraform-providers/terraform-provider-aws/issues/11037

Work-around to get this working is to pin the provider version as so:

```
terraform {
  required_providers {
    aws = "2.39.0"
  }
}
```

**Comment by whummer** [comment]
Thanks for reporting @ciaranevans - looks like this could be a duplicate of #1822, should be fixed in #1838 . Can you please give it a try with the latest version of the Docker image? Thanks

**Comment by ciaranevans** [comment]
Yep, fixed!
